src/context.h

```diff
@@ -131,6 +131,8 @@ typedef struct getdns_upstream {
 	/* How is this upstream doing over UDP? */
 	int                      to_retry;
 	int                      back_off;
+	size_t                   udp_responses;
+	size_t                   udp_timeouts;
```

Comment:
I have another pull request on the way where I am touching call reporting for TCP/TLS so will include it there.